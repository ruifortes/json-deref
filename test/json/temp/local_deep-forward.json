{
  "a": 1,
  "b": {
    "a": {"$ref": "#c"},
    "b": {
      "a": "bba value",
      "b": "bbb value"
    },
    "c": "bc value",
    "d": {"$ref": "#c/a"}
  },
  "c": {
    "a": 123,
    "b": {"$ref": "#d"}
  },
  "d": {"$ref": "#b/d"}
}
