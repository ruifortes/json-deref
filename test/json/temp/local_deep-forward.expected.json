{
  "a": 1,
  "b": {
    "a": {
      "a":123,
      "b":123
    },
    "b": {
      "a": "bba value",
      "b": "bbb value"
    },
    "c": "bc value",
    "d": 123
  },
  "c": {
    "a": 123,
    "b": 123
  },
  "d": 123
}
